spec.md

```diff
@@ -928,7 +928,7 @@ of parentheses, spaces, and braces.
 <?php
 
 do {
-    // structure body;
+    // Structure body;
```

Comment:
This PR leaves semicolons in some places and removes them from others, we should probably normalize those as well.

Reply:
Agreed. Oversight on my part.

Fixed.